readme.md

```diff
@@ -3,14 +3,26 @@ LÖVE is an *awesome* framework you can use to make 2D games in Lua. It's free,
 [![Build Status: Windows](https://ci.appveyor.com/api/projects/status/chc0hdr08wv1d5c7?svg=true)](https://ci.appveyor.com/project/AlexSzpakowski/love)
 [![Build Status: Github CI](https://github.com/love2d/love/workflows/continuous-integration/badge.svg)](https://github.com/love2d/love/actions?query=workflow%3Acontinuous-integration)
 
-Documentation
--------------
+Please see our [living wiki documentationwiki][wiki] for deeper explanations and documentation of this framework.
```

Comment:
I don't think it's necessary to remove the Documentation section, there's also a typo here and "living" is redundant when talking about a wiki. I think the Documentation section was fine as-is.

Reply:
Right. I ended up canceling out this change so both headline and text stays unchanged.